extensions/ql-vscode/src/cli.ts

```diff
@@ -43,6 +43,16 @@ export interface QuerySetup {
   compilationCache?: string;
 }
 
+/**
+ * The expected output of `codeql resolve queries --format bylanguage`.
+ */
+export interface QueryInfoByLanguage {
+  // What should the value of these objects be? I've only ever seen it as an empty object.
```

Comment:
Yeah, it's currently just a placeholder and is only ever an empty object. If `unknown` below is enough for the JSON deserializer to be happy then that seems as good a placeholder as any for the moment.

Reply:
Great, thank you! (And thanks for the review in general ⚡ ) 

I'll convert this back to draft while I tidy up the rest 😀